playground/providers.ts

```diff
@@ -59,24 +59,34 @@ export const providers: Provider[] = [
       }
     ]
   },
-  // Sanity
+  // Fastly
   {
-    name: 'sanity',
+    name: 'fastly',
+    samples: [
+      { src: '/image.jpg' },
+      { src: '/plant.jpeg' }
+    ]
+  },
+  // Prismic
+  {
+    name: 'prismic',
     samples: [
       {
-        src: 'image-7aa06723bb01a7a79055b6d6f5be80329a0e5b58-780x1170-jpg'
+        src: 'https://images.prismic.io/200629-sms-hoy/f596a543-d593-4296-9abd-3d3ac15f1e39_ray-hennessy-mpw37yXc_WQ-unsplash.jpg?auto=compress,format&w=600&h=900'
       },
       {
-        src: 'image-7aa06723bb01a7a79055b6d6f5be80329a0e5b58-780x1170-jpg',
-        width: 200,
-        height: 200,
-        fit: 'crop'
+        src: 'https://images.prismic.io/200629-sms-hoy/f596a543-d593-4296-9abd-3d3ac15f1e39_ray-hennessy-mpw37yXc_WQ-unsplash.jpg?auto=compress,format&w=600&h=900',
```

Comment:
is there a reason to add these via query strings rather than via modifiers directly on `<nuxt-img>` (both here and throughout the PR)?